src/test/java/unstable/PanelFocusTest.java

```diff
@@ -42,7 +40,9 @@ public void setupMethod() {
         panels.add(test3);
 
         globalConfig.setPanelInfo(panels);
-        configFileHandler.saveGlobalConfig(globalConfig);
+        ConfigFileHandler configFileHandler =
+                new ConfigFileHandler(Preferences.DIRECTORY, Preferences.TEST_CONFIG_FILE, globalConfig);
```

Comment:
Also need to rename `TEST_CONFIG_FILE` to `TEST_GLOBAL_CONFIG_FILE` or something along the way.